changelog/unreleased/kong/fix_dns_enable_dns_no_sync.yml

```diff
@@ -1,3 +0,0 @@
-message: The default value of `dns_no_sync` option has been changed to `on`
```

Comment:
Is this PR going to be released at 3.5? I'm wondering why this file is being deleted.

Reply:
yes. This patch try to fix dns blocking issue, so we need to revert original temporary fix that changes  `dyns_no_sycn` option to `on`

Reply:
Sry, my mistake, this patch will not be merged into 3.5.0.0, because it's already frozen.
But it could be involved in 3.5.x.x.

Reply:
If you're trying to ship it in patch release of 3.5.0, please restore this changelog file.